client/src/pages/Explorer/Explore.js

```diff
@@ -102,8 +110,14 @@ const Explore = () => {
             city.favorite = favoriteList.indexOf(city.name) >= 0;
             return city;
           });
-          const updatedList = getUpdatedList(cityList, favoriteList);
-          setPlaces(updatedList);
+
+          const pathName = window.location.pathname;
+          if (pathName === "/profile/favoritedestinations") {
+            renderProfileFavPage(cityList, favoriteList);
+          } else {
+            const updatedList = getUpdatedList(cityList, favoriteList);
+            setPlaces(updatedList);
+          }
```

Comment:
this is not as bad as I thought, though it still seems hacky. I'm undecided on whether or not it's better to just have a separate component. For now though we can roll with it